webextensions/api/webRequest.json

```diff
@@ -1541,9 +1544,10 @@
               },
               "edge": {
                 "notes": [
-                  "Asynchronous event listeners are not supported."
+                  "Asynchronous event listeners are not supported.",
+                  "`requestBody` is not supported."
```

Comment:
Should use `<code>requestBody</code>`, or just plaintext. Markdown is not supported in notes (yes I know lots of existing notes use it :-/